test/conformance/service_test.go

```diff
@@ -536,4 +536,47 @@ func TestReleaseService(t *testing.T) {
 	}
 }
 
+func TestMultipleNamespace(t *testing.T) {
+	t.Parallel()
+
+	altServiceNamespace := fmt.Sprintf("%s-%s", test.ServingNamespace, test.ObjectNameForTest(t))
+	serviceName := test.ObjectNameForTest(t)
+
+	defaultClients := e2e.Setup(t) // This one uses the default namespace `test.ServingNamespace`
+	altClients := e2e.SetupWithNamespace(t, altServiceNamespace)
+
+	_, err := test.CreateNamespace(altClients, altServiceNamespace)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer test.DeleteNamespace(altClients, altServiceNamespace)
+
+	defaultResources := test.ResourceNames{
+		Service: serviceName,
+		Image:   pizzaPlanet1,
+	}
+	if _, err := test.CreateRunLatestServiceReady(t, defaultClients, &defaultResources, &test.Options{}); err != nil {
```

Comment:
To save runtime, we could do creations in parallel.

Reply:
I actually experimented with that; it didn't look as clean and it didn't take too long to spin up the service synchronously.